src/components/itemModal.html

```diff
@@ -65,9 +65,12 @@
                                         data-bind="if: (player.alreadyCaughtPokemon($data.name))">
                                         <button class="btn btn-secondary smallButton list-group-item-action" style="padding-right:15px"
                                                 data-bind="click: function() {ItemHandler.pokemonSelected(name)}, css:{ 'pokemon-selected': ItemHandler.pokemonSelected() === name}">
-                                            <img class="smallImage"
-                                                 data-bind="attr:{ src: '/assets/images/pokemon/' + id +'.png' }"
-                                                 src=""/>
+                                            <span data-bind="if: player.alreadyCaughtPokemonShiny($data.name)">
+                                                <img class="smallImage" data-bind="attr:{ src: '/assets/images/shinypokemon/' + id +'.png' }"/>
```

Comment:
How about this instead of the `if/ifnot`?
```suggestion
                                                <img class="smallImage" data-bind="attr:{ src: '/assets/images/' + (player.alreadyCaughtPokemonShiny($data.name) ? 'shiny' : '') + 'pokemon/' + id +'.png' }"/>
```